pkg/kubelet/dockertools/docker_manager.go

```diff
@@ -1832,23 +1831,67 @@ func addNDotsOption(resolvFilePath string) error {
 		return fmt.Errorf("ResolvConfPath %q does not exist", resolvFilePath)
 	}
 
-	glog.V(4).Infof("DNS ResolvConfPath exists: %s. Will attempt to add ndots option: %s", resolvFilePath, ndotsDNSOption)
+	curResolvFileContent, err := getFileContent(resolvFilePath)
```

Comment:
Added this only in the sake of debugging/troubleshooting to provide info log: which was the content and how it was replaced. But, yeah, having this doesn't make much sense, will remove it.